src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -326,3 +347,27 @@ def load_arguments(self, _):
 
     with self.argument_context('appconfig replica create') as c:
         c.argument('location', arg_type=get_location_type(self.cli_ctx), help='Location at which to create the replica.')
+
+    with self.argument_context('appconfig snapshot create') as c:
+        c.argument('snapshot_name', arg_type=snapshot_name_arg_type)
+        c.argument('filters', arg_type=snapshot_filter_arg_type)
+        c.argument('composition_type', options_list=['--composition-type'], arg_type=get_enum_type(["all", "group_by_key"]), help='Composition type used in building app configuration snapshots.')
+        c.argument('retention_period', options_list=['--retention-period'], type=int, help='Duration in seconds for which a snapshot can remain archived before expiry. If not specified, retention period defaults to the equivalent of 30 days. (2592000s)')
+        c.argument('tags', arg_type=tags_type, help="Space-separated tags: key[=value] [key[=value] ...].")
```

Comment:
it's only 30 days for standard tier. 7 days for free. I'd omit the actual value and just mention it's optional.

Reply:
Got it, however, I would want to know if there is an upper limit for the number of days a snapshot can be retained.

Reply:
Upper limit is 7 days for free store, 90 for standard.

Reply:
I am not sure about this but would it be worth mentioning the limits as is done for config store retention period as shown below? 
<img width="761" alt="image" src="https://user-images.githubusercontent.com/102393878/207191892-0327ae99-08e2-4832-abfc-4c3fc18f5e2c.png">

Reply:
I don't see a problem with adding it, but needs to mention the difference between standard and free.

Reply:
Got it. The help message has been updated accordingly.